### DISCUSSION TITLE
Can one convert a Creditor(Expense)/Debtor(Revenue) to a banking account(Asset) ?

### DISCUSSION THREAD

**Discussion by martijn-gr** [firefly-iii/firefly-iii#4662] [Firefly III] [answered]
Hi Folks,

I have seen several posts where one could not create a bank account because of the IBAN was already in use.
The suggested solution was often to throw away the history, but this will not solve the issue because often some reference to the IBAN remains.

Background to the why:
I have a personal financial administrator that has several account in it, that has transferred in the past money to some account that I now would like to become part of the financial administration.

TL;DR:
Is there any way to release an IBAN to re-use it on a banking account or by converting a Creditor(Expense)/Debtor(Revenue) to a banking account(Asset)?

**Comment by JC5** [comment, marked as answer]
If you remove the IBAN from the account and put it in the notes or something, it's freed up to be used in other accounts.

**Reply by martijn-gr** [reply]
Thanks for your quick answer.
Although it was indeed possible to remove the IBAN frome the Expenses account, it was impossible to add it as an Asset account.

Reason: IBAN was already in use somewhere.

Now luckily I was still a little bit awake, quickly checked Revenue accounts and found it in there also.

So for others finding this topic, if you can't add an asset account make sure you check both Expense and Revenue accounts for the IBAN.
